idseq_pipeline/commands/non_host_alignment_functions.py

```diff
@@ -39,11 +39,15 @@
 ROOT_DIR = '/mnt'
 DEST_DIR = ROOT_DIR + '/idseq/data' # generated data go here
 REF_DIR = ROOT_DIR + '/idseq/ref' # referene genome / ref databases go here
-
+# input files
+EXTRACT_UNMAPPED_FROM_BOWTIE_SAM_OUT1 = 'unmapped.bowtie2.lzw.cdhitdup.priceseqfilter.unmapped.star.1.fasta'
```

Comment:
Why is this duplicated in two different files; not related to your change, but seems like this stuff belongs into common.py (for now, until we make a special file for it).

Reply:
i dont understand either. i was very confused with it.